Domain-Setup.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Obtain A Domain Name and Redirect Domain DNS
+layout: topic
+categories: guide
+---
+
+Now that you have a webpage and are ready to deploy it, you need to 
+set up a domain name for your website. This wiki will walk you through 
+that process.
+
+## Step 1: Obtain a Domain Name
+
+It is recommended to get a domain name off of [NameCheap](https://www.namecheap.com), 
+but you can purchase a domain name from other providers (including 
+[GoDaddy](https://www.godaddy.com/) and [HostGator](https://www.hostgator.com/domains)).
+
+To get your domain name:
+- Create an account with the provider of your choosing.
+- Pick a name that sounds good to you. NameCheap has convenient search 
+  functionality to help you find the perfect domain name for your 
+  project.
+- **For NameCheap:** select [WhoIsGuard](https://www.namecheap.com/support/knowledgebase/article.aspx/278/37/what-is-whoisguard) 
```

Comment:
One slight note on this, WhoIsGuard isn't available on all TLDs. My .us domain doesn't allow for it because of Nexus requirements to verify I live in the US. [https://www.namecheap.com/support/knowledgebase/article.aspx/280/37/do-you-provide-privacy-protection-for-all-registered-domains](https://www.namecheap.com/support/knowledgebase/article.aspx/280/37/do-you-provide-privacy-protection-for-all-registered-domains)

Reply:
Cool, that's good to know. I'll add wording saying "if available" or something similar. Thanks @dsdude

Reply:
Addressed in d37e4b7fa52a0f04e55dffc0e103bda1ac99adf6